packages/playwright/src/reporters/base.ts

```diff
@@ -489,11 +489,34 @@ export function stripAnsiEscapes(str: string): string {
   return str.replace(ansiRegex, '');
 }
 
+function characterWidth(c: string) {
+  return /[\u4e00-\u9fff]/.test(c) ? 2 : 1;
```

Comment:
This range only covers the CJK, but there are more double-width characters, but there is more. https://www.npmjs.com/package/east-asian-width would be a proper way of handling this.

Reply:
was hesitant to introduce a dependency so I proposed this shortcut. agree that a dependency is good. I added `get-east-asian-width` in https://github.com/microsoft/playwright/pull/33322/commits/3d713aaf536d9a5ad3dd625db606b7525ace7342, it has ~7 million monthly downloads.